src/components/Identification/index.js

```diff
@@ -50,6 +51,7 @@ export const Identification = props => {
   const [snackbarKey, setSnackbarKey] = useState(0); // A key to make every snackbar display unique
   const [snackbarType, setSnackbarType] = useState(''); // Either success or error
   const [userProfile, setUserProfile] = useState(null);
+  const [currentWidth, setCurrentWidth] = useState(null);
   const cropper = useRef();
```

Comment:
Used for determining the current width of the screen based on Material-UI's breakpoints. This is used to determine the text to display and item sizing in the Photo Updater Dialog box.